Makefile

```diff
@@ -18,294 +19,140 @@ ifeq ($(wildcard $(CONFIG_FILE)),)
-	@cd $@
-
-examples/datadog-api-client-ruby:
-	@git clone --depth 1 --branch $(BRANCH) https://github.com/DataDog/datadog-api-client-ruby.git $@ || git clone --depth 1 --branch $(shell grep -A1 "datadog-api-client-ruby" data/sdk_versions.json | grep version | cut -f 2 -d ':' | tr -d '"') https://github.com/DataDog/datadog-api-client-ruby.git $@
-	@cd $@
-
-examples/datadog-api-client-typescript:
-	@git clone --depth 1 --branch $(BRANCH) https://github.com/DataDog/datadog-api-client-typescript.git $@ || git clone --depth 1 --branch $(shell grep -A1 "datadog-api-client-typescript" data/sdk_versions.json | grep version | cut -f 2 -d ':' | tr -d '"') https://github.com/DataDog/datadog-api-client-typescript.git $@
-	@cd $@
-
-.PHONY: examples/go examples/java examples/python examples/ruby examples/typescript examples
-
 EXAMPLES_DIR = $(shell pwd)/examples/content/en/api
-
-examples/go: examples/datadog-api-client-go clean-go-examples
-	-cp -Rn examples/datadog-api-client-go/examples/v* ./content/en/api/
-
-examples/java: examples/datadog-api-client-java clean-java-examples
-	-cp -Rn examples/datadog-api-client-java/examples/v* ./content/en/api/
-
-examples/python: examples/datadog-api-client-python clean-python-examples
-	-find examples/datadog-api-client-python/examples -iname \*.py -exec mv {} {}beta \;
-	-cp -Rn examples/datadog-api-client-python/examples/v* ./content/en/api
-
-examples/ruby: examples/datadog-api-client-ruby clean-ruby-examples
-	-find examples/datadog-api-client-ruby/examples -iname \*.rb -exec mv {} {}beta \;
-	-cp -Rn examples/datadog-api-client-ruby/examples/v* ./content/en/api
-
-examples/typescript: examples/datadog-api-client-typescript clean-typescript-examples
-	-cp -Rn examples/datadog-api-client-typescript/examples/v* ./content/en/api
-
-examples: examples/go examples/java examples/python examples/ruby examples/typescript ## Build the examples.
-
-start-docker: clean ## Start server in a docker container.
+EXAMPLES_REPOS := datadog-api-client-go datadog-api-client-java datadog-api-client-python datadog-api-client-ruby datadog-api-client-typescript
+
+# Set defaults when no makefile.config or missing entries
+# Use DATADOG_API_KEY if set, otherwise try DD_API_KEY and lastly fall back to false
+GITHUB_TOKEN ?= ""
+DD_API_KEY ?= false
+DD_APP_KEY ?= false
+DATADOG_API_KEY ?= $(DD_API_KEY)
+DATADOG_APP_KEY ?= $(DD_APP_KEY)
+CONFIGURATION_FILE ?= "./local/bin/py/build/configurations/pull_config_preview.yaml"
+
+
+clean-all: clean clean-examples
+	rm -rf ./node_modules ./hugpython ./public ./integrations_data
+
+# remove build generated content
+# removing only git ignored files
+clean:
+	@git clean -Xf ./content
+	@git clean -Xf ./data
+	@git clean -Xf ./static/images/marketplace
```

Comment:
I think at this point its safe to remove, it was an unintended file which we deleted from history but kept some automation in case it crept back in during cleanup.